### DISCUSSION TITLE
Automatic release notes

### DISCUSSION THREAD

**Discussion by LuckeeDev** [jscutlery/semver#460] [Q&A] [answered]
Is there any way to associate the release changelog generated by semver to its corresponding GitHub release? Is there any GitHub Action that does this?

**Comment by edbzn** [comment, marked as answer]
@LuckeeDev you can use the github executor as a post target, that's what we use in this repo. You can include the whole changelog or only the notes related to the new version. More info [here](https://github.com/jscutlery/semver/blob/main/packages/semver/src/executors/github/README.md).
